packages/flutter_test/test/live_binding_test.dart

```diff
@@ -16,4 +18,93 @@ void main() {
     // starts with a PointerAddedEvent.
     await gesture.addPointer();
   });
+
+  binding.framePolicy = LiveTestWidgetsFlutterBindingFramePolicy.fullyLive;
+  int standardFrameCount = 0;
+  testWidgets('Measure refresh rate', (WidgetTester tester) async {
+    await tester.pumpWidget(const Text(
+        'foo',
+        textDirection: TextDirection.ltr,
+      ));
+      final AnimationController test = AnimationController(
+        duration: const Duration(seconds: 3),
+        vsync: tester,
+      );
+
+      test.forward();
+      final Ticker ticker = tester.createTicker((Duration elapsed) {
+        standardFrameCount ++;
+      });
+      ticker.start();
+      await tester.binding.delayed(const Duration(seconds: 2));
+      ticker.stop();
+      ticker.dispose();
+      test.dispose();
+  });
+
+  binding.framePolicy = LiveTestWidgetsFlutterBindingFramePolicy.benchmarkLive;
+  testWidgets('Frame Counter and Input Delay for benchmarkLive', (WidgetTester tester) async {
+    await tester.pumpWidget(MaterialApp(
+      home: Scaffold(
+        body: Scrollbar(
+          child: ListView(
+            children: <Widget>[
+              for (int n = 0; n < 100; n++)
+                Container(height: 40.0, child: Text('$n')),
+            ],
+          ),
+        ),
+      ),
+    ));
+    await tester.pumpAndSettle();
+    final Offset location = tester.getCenter(find.byType(ListView));
+    int frameCount = 0;
+    tester.binding.addPersistentFrameCallback((Duration elapsed) {
+      frameCount += 1;
+    });
+
+    const int timeInSecond = 2;
+    const Duration totalTime = Duration(seconds: timeInSecond);
+    const int moveEventNumber = timeInSecond * 120;  // 120Hz
+    const Offset movePerRun = Offset(0.0, -400.0 / moveEventNumber);
+    final List<PointerEventRecord> records = <PointerEventRecord>[
+      PointerEventRecord(Duration.zero, <PointerEvent>[
+        PointerAddedEvent(
+          timeStamp: Duration.zero,
+          position: location,
+        ),
+        PointerDownEvent(
+          timeStamp: Duration.zero,
+          position: location,
+          pointer: 1,
+        ),
+      ]),
+      ...<PointerEventRecord>[
+        for (int t=0; t < moveEventNumber; t++)
+          PointerEventRecord(totalTime * (t / moveEventNumber), <PointerEvent>[
+            PointerMoveEvent(
+              timeStamp: totalTime * (t / moveEventNumber),
+              position: location + movePerRun * t.toDouble(),
+              pointer: 1,
+              delta: movePerRun,
+            )
+          ])
+      ],
+      PointerEventRecord(totalTime, <PointerEvent>[
+        PointerUpEvent(
+          timeStamp: totalTime,
+          position: location + movePerRun * moveEventNumber.toDouble(),
+          pointer: 1,
+        )
+      ])
+    ];
+    final List<Duration> delays = await tester.handlePointerEventRecord(records);
+
+    // This will typically be 113 +- 1 for fullyLive flag.
```

Comment:
What's the number here for other flags, on host unit tests and on device?

Reply:
All other flags don't response to frame requests from the engine, including the benchmark, and they basically don't track animation unless manually pump-ed. So these numbers are not relevant for other flags. 
As for the number on host and on device, that's the issue why I'm getting flaky result. The number here comes from my local test on my macbook, and it seems on Cirrus they get different number, varying from 60+- ~90+-. Is there better place to test these number?